kimchi/src/circuits/constraints.rs

```diff
@@ -330,6 +381,13 @@ impl<F: PrimeField + SquareRootField> Builder<F> {
         self
     }
 
+    /// Set up the foreign field modulus.
+    /// If not invoked, it is `vec![]` by default.
+    pub fn foreign_field_modulus(mut self, foreign_field_modulus: Vec<F>) -> Self {
+        self.foreign_field_modulus = foreign_field_modulus;
```

Comment:
btw should we enforce anything on this foreign modulus? We probably need to check that it's bigger than our own modulus at the very least, and also that it can fit in 3*88 bits?

Reply:
Ok, I agree.